www/src/scss/viewer/verse-slide/_verse-slide.scss

```diff
@@ -71,6 +71,10 @@
   width: 18%;
   max-width: 150px;
   z-index: 2;
+
+  &.hide-quicktools {
+    left: -18%;
```

Comment:
I think percent (%) will create an issue if screen size changes. It should be changed to the pixel(px) that is the width of quick tool panel.

Reply:
Yes veerji, thanks for pointing it out. I will change this one.

Reply:
It's fine then :+1: